content/articles/simulating-a-simple-epidemic-spreading-process-on-graphs/Simulating a Simple Epidemic Spreaging Process on  40a38af8bf6a4743a874f9bdcc487a61.md

```diff
@@ -392,6 +410,8 @@ function spreading_plot!(G, r)
 end
```

Comment:
The code above isn't properly formated. Please check other code blocks that need formatting and edit them.